src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -0,0 +1,100 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { VerifiedIcon } from 'nft/components/icons'
+import { CollectionInfoForAsset, GenieAsset } from 'nft/types'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+
+const MAX_WIDTH = 560
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-height: ${({ theme }) => `calc(100vh - ${theme.navHeight}px)`};
```

Comment:
on smaller screens there are navbars on the top and bottom so navHeight will need to be subtracted twice

Reply:
![Screen Shot 2023-04-27 at 09 20 14 ](https://user-images.githubusercontent.com/11512321/234926849-e37999a8-33cf-41fe-a623-ae6f6dc0c834.png)